decompiler/pipeline/controlflowanalysis/readability_based_refinement.py

```diff
@@ -171,22 +171,29 @@ def _single_defininition_reaches_node(ast: AbstractSyntaxTree, variable_init: As
     return True
 
 
-def _separated_by_loop_node(init_node: AbstractSyntaxTreeNode, usage_node: AbstractSyntaxTreeNode) -> bool:
```

Comment:
I don't think that the refactoring of readability-based-refinements should be in this PR addressing the networkx changes

Reply:
was rebased on the wrong branch. Is fixed :)